src/plugin/Manager.cc

```diff
@@ -140,6 +141,32 @@ void Manager::SearchDynamicPlugins(const std::string& dir)
 	closedir(d);
 	}
 
+void Manager::LoadScriptsForStaticPlugins()
```

Comment:
Yeah, I think we can completely get rid of this function.  I could generate a plugin load script in addition to the preload one that is already being generated to get the runtime loading out of the code.  Originally the preload was done in that method too, but it turned out not to be feasible because of dependency ordering issues that I fixed yesterday.

And I think that the ZEEKPATH could use a similar strategy since none of that would be expected to change post build.